vumi/connectors.py

```diff
@@ -162,3 +163,28 @@ def _ignore_message(self, failure, msg):
         return self.publish_event(TransportEvent(
             user_message_id=msg['message_id'], nack_reason=str(failure.value),
             event_type='nack'))
+
+
+class PublishStatusConnector(BaseConnector):
+    @inlineCallbacks
+    def setup(self):
+        yield self._setup_publisher('status')
+
+    def publish_status(self, msg, endpoint_name=None):
+        return self._publish_message('status', msg, endpoint_name)
```

Comment:
Is there a point to allowing an endpoint to be set when calling `publish_status`?